aiosonic/__init__.py

```diff
@@ -222,11 +222,11 @@ async def read_chunks(self) -> AsyncIterator[bytes]:
 
     def __del__(self):
         # clean it
-        if self.chunked and not self.chunks_readed:
-            loop = None
-            if self.connection:
-                loop = get_loop()
-                loop.create_task(self.connection.release())
+        if self.connection and self.connection.blocked:
+            if self.connection.writer:
+               self.connection.writer._transport.abort()
+            self.connection.blocked = False
+            self.connection.connector.pool.release(self.connection)
```

Comment:
I don't like to call the pool.release directly but for now it is ok, gonna try to refactor it

Reply:
Both CyclicQueuePool and SmartPool now have non-async release methods. We could convert the whole chain of release() inside Connection class and Connector class to non-async perhaps? So we can call release on the Connection from a blocking context.

Reply:
Yes, I'm working on a PR just now https://github.com/sonic182/aiosonic/pull/457